program.css

```diff
@@ -0,0 +1,155 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@1,800&family=Raleway:wght@100;200;300;400&display=swap');
```

Comment:
Double check which fonts are included in the nav.css and footer.css, I think Raleway might already be included

Reply:
I changed it to import only the Raleway weights that aren't already included.